docs/shared/features/powerpack/custom-caching.md

```diff
@@ -1 +1,33 @@
 # Change the Remote Cache Storage Location
```

Comment:
Can we have the intro be similar to the other feature pages, where it clearly highlights the benefits right away, maybe with a couple of bullet points?

Like
- https://nx.dev/features/cache-task-results
- https://nx.dev/features/generate-code
- ...